api-tests/tests/api-spec.js

```diff
@@ -4,8 +4,8 @@ var base_url = "http://api:8080/";
 describe("When testing 'api/pet'", function(){
     it("should respond with the URL of a cat GIF", function(done) {
         request(base_url + 'api/pet', function(error, response, body){
-            expect(body).toMatch(/\{"imageId":\d+,"url":"http:/);
+            expect(body).toMatch(/\{"imageId":\d+,"url":"https:/);
```

Comment:
The better way to fix this would be to allow both `http` and `https` `toMatch` with something like:

```
            expect(body).toMatch(/\{"imageId":\d+,"url":"http(s?):/);
```